apps/rush/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Change Log - @microsoft/rush
 
-This log was last generated on Fri, 21 Jul 2017 22:34:44 GMT and should not be manually modified.
+This log was last generated on Wed, 30 Aug 2017 18:46:19 GMT and should not be manually modified.
+
+## 3.0.15
+Wed, 30 Aug 2017 18:24:39 GMT
+
+### Patches
+
+- Add tar as a dependency of Rush which is required work for replacing the temp_modules/*/package.json files with TGZ files
+- Add repositoryUrl to RushConfiguration to track remote repository
+- Use the new Json API from node-core-library
+- Add two new methods to ChangeFile class
+- Introduce an experimental "rush version" action to manage project versions based on version policy
+- Make "rush generate" not throw if there is a problem reading the shrinkwrap file
```

Comment:
>throw [](start = 27, length = 5)

Throwing an exception is something code does.  This changelog is for a tool.  Something like:

"Fix an issue where "rush generate" could not recover from a corrupted shrinkwrap file"

@nickpape-msft

Reply:
Yes, that would be even better.

---
In reply to: [136168243](https://github.com/Microsoft/web-build-tools/pull/338#discussion_r136168243) [](ancestors = 136168243)